entrypoint.js

```diff
@@ -1,28 +1,36 @@
 const axios = require('axios');
-const message = require('./message.js');
+const _ = require('lodash');
+const core = require('@actions/core');
+const message = require('./message');
 
 const REQUIRED_ENV_VARS = [
   "GITHUB_EVENT_PATH",
   "SLACK_WEBHOOK",
 ];
 
-REQUIRED_ENV_VARS.forEach(env => {
-  if (!process.env[env] || !process.env[env].length) {
-    console.error(`Missing environment variable. ${env} is required.`);
+try {
+  _.forEach(REQUIRED_ENV_VARS, env => {
+    if (_.isEmpty(process.env[env])) {
+      process.exitCode = 1;
+      throw new Error(`Missing environment variable. ${env} is required.`);
+    }
+  });
+} catch (e) { core.setFailed(e.message); }
 
-    return process.exit(1);
-  }
-});
 
-console.log("Sending message ...");
-(() => axios
-  .post(process.env.SLACK_WEBHOOK, message.get())
-  .then(() => {
-    console.log("Message sent ! Shutting down ...");
-    return process.exitCode = 0;
-  })
-  .catch(err => {
-    console.error("Message :", err.response ? err.response.data : err.message);
-    return process.exitCode = 1;
-  })
-)();
+(() => {
+  if (!process.exitCode) {
+    core.info("Sending message ...");
+
+    axios.post(process.env.SLACK_WEBHOOK, message.get())
+      .then(() => {
+        process.exitCode = 0;
+        return core.info('Message sent! Shutting down ...');
+      })
+      .catch((err) => {
+        process.exitCode = 1;
+        const errMessage = err.response ? err.response.data : err.message;
+        return core.setFailed(`Error: ${errMessage}`);
+      })
+  }
+})();
```

Comment:
```suggestion
if (!process.exitCode) {
  core.info("Sending message ...");

  axios.post(process.env.SLACK_WEBHOOK, message.get())
    .then(() => {
      process.exitCode = 0;
      return core.info('Message sent! Shutting down ...');
    })
    .catch((err) => {
      process.exitCode = 1;
      const errMessage = err.response ? err.response.data : err.message;
      return core.setFailed(`Error: ${errMessage}`);
    })
}
```

I guess we can remove this `(() => { /* code... */ })()` thing as I primarily used it to make the POST call with a top-level `async`/`await` handling.